tests/extensions/aria_extension_tosca/simple_v1_0/templates/common/test_template_interface.py

```diff
@@ -0,0 +1,914 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not sure providing those macros helps (apart from code reuse, it's really hard to understand and write new tests).